test/libethereum/TransactionTests.cpp

```diff
@@ -35,7 +35,7 @@ namespace dev {  namespace test {
 
 void doTransactionTests(json_spirit::mValue& _v, bool _fillin)
 {
-	unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(Network::FrontierTest)).createSealEngine());
+	unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(Options::get().sealEngine)).createSealEngine());
```

Comment:
Also change a line in BlockChainHelper.cpp line 444
that would be enough to make all* blockchain tests to run on defined sealengine